app/save-and-restore/ui/pom.xml

```diff
@@ -120,6 +120,16 @@
 			<version>5.1.7.RELEASE</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
See https://github.com/ControlSystemStudio/phoebus/blob/master/dependencies/README.md
In short, when you think you have to add another dependency, also add it to dependencies/phoebus-target/pom.xml

Reply:
Thank you for your feedback. I didn't know there are guidelines for dependencies.

Reply:
I'm not sure if I understand the instructions correctly. A dependency needs to be added to both `dependencies/phoebus-target/pom.xml` and a submodule `pom.xml`?

Reply:
Yes. To dependencies/phoebus-target/pom.xml so we have a list in one place of all external dependencies. The result is also a 'lib' folder that can be used to build with ant or Eclipse or ...  For maven, you also need to add it to the submodule pom.